force-app/main/default/chat/lwc/messageInput/messageInput.js

```diff
@@ -0,0 +1,26 @@
+//todo:: const labels in html
+import {LightningElement} from 'lwc';
+
+const SEND_EVENT = 'send';
```

Comment:
SEND_EVENT -> SEND_EVENT_NAME

Reply:
fixed in 8d870610e229cc8a420aa5daeddcdb7010ca08a5